tests/sentry/models/test_project.py

```diff
@@ -407,6 +409,55 @@ def test_remove_team_clears_alerts(self):
         assert alert_rule.user_id is None
 
 
+class ProjectSettings(TestCase):
+    def setUp(self):
```

Comment:
Could you add a docstring that just says something to the effect of "`ProjectOption` trumps `ProjecTemplateOption`? It was a little hard to understand at first when reading the tests.